### ISSUE TITLE
Model API knowledge

### ISSUE THREAD

**Issue by eas4ai** [openai/openai-openapi#178]
I know this isn't really an API issue... but, if I try to use any of the models to code an app using the API, I have to train it on the current API. Its knowledge is restricted to an old version. This wastes so much context and tokens it's crazy. I don't see a down side for OpenAI to train the models on the current API except that it gives the model self context. However, as a developer trying to use the model to code using the API it's extremely frustrating.

**Comment by markstuart-oai** [comment]
Agreed. Having to repeatedly supply current API docs is frustrating. Models' built-in knowledge has a cutoff and doesn’t automatically update with API changes.  Even a recent cutoff doesn’t guarantee correct code.

Our [Docs MCP server](https://developers.openai.com/learn/docs-mcp) lets compatible coding tools look up current documentation, though those lookups still use context.

This is valid feedback about model quality, rather than something we can address in the OpenAPI specification.
